vmaas/common/rpm_utils.py

```diff
@@ -81,7 +81,6 @@ def rpmver2array(rpm_version: str) -> list:
     arr.append((-2, ''))          # fill array to "n_len"
     return arr
 
-
```

Comment:
That's some pep standard to have 2 empty lines between functions. Please, do not change files if it's not needed.

Reply:
I just removed strip_prefixes from there. Corrected because there was extra line by mistake I guess.